astroquery/gaia/core.py

```diff
@@ -218,44 +217,53 @@ def load_data(self, ids, *, data_release=None, data_structure='INDIVIDUAL', retr
             the Datalink items tags will not be checked.
         format : str, optional, default 'votable'
-            loading format. Other available formats are 'csv', 'ecsv','votable_plain' and 'fits'
-        output_file : string or pathlib.PosixPath, optional, default None
-            file where the results are saved.
-            If it is not provided, the http response contents are returned.
+            loading format. Other available formats are 'csv', 'ecsv','votable_plain', 'json' and 'fits'
+        dump_to_file: boolean, optional, default False.
+            If it is true, a compressed directory named "datalink_output_<time_stamp>.zip" with all the DataLink
+            files is made in the current working directory. The <time_stamp> format follows the ISO 8601 standard:
+            "yyyymmddThhmmss".
         overwrite_output_file : boolean, optional, default False
-            To overwrite the output_file if it already exists.
+            To overwrite the output file ("datalink_output.zip") if it already exists.
         verbose : bool, optional, default 'False'
             flag to display information about the process
 
         Returns
         -------
         A dictionary where the keys are the file names and its value is a list of astropy.table.table.Table objects
         """
-        now = datetime.now(timezone.utc)
-        now_formatted = now.strftime("%Y%m%d_%H%M%S")
-        temp_dirname = "temp_" + now_formatted
-        downloadname_formated = "download_" + now_formatted
 
         output_file_specified = False
-        if output_file is None:
+
+        now = datetime.datetime.now(datetime.timezone.utc)
+        if not dump_to_file:
+            now_formatted = now.strftime("%Y%m%d_%H%M%S")
+            temp_dirname = "temp_" + now_formatted
+            downloadname_formated = "download_" + now_formatted
             output_file = os.path.join(os.getcwd(), temp_dirname, downloadname_formated)
+
         else:
+            output_file = 'datalink_output_' + now.strftime("%Y%m%dT%H%M%S") + '.zip'
             output_file_specified = True
-
-            if isinstance(output_file, str):
-                if not output_file.lower().endswith('.zip'):
-                    output_file = output_file + '.zip'
-            elif isinstance(output_file, Path):
-                if not output_file.suffix.endswith('.zip'):
-                    output_file.with_suffix('.zip')
-
             output_file = os.path.abspath(output_file)
+            log.info(f"DataLink products will be stored in the {output_file} file")
+
             if not overwrite_output_file and os.path.exists(output_file):
                 raise ValueError(f"{output_file} file already exists. Please use overwrite_output_file='True' to "
                                  f"overwrite output file.")
 
         path = os.path.dirname(output_file)
 
+        log.debug(f"Directory where the data will be saved: {path}")
+
+        if path != '':
```

Comment:
This maybe cleaner as `''` will be False here:

```suggestion
        if path:
```